assets/common.sh

```diff
@@ -79,6 +79,26 @@ setup_kubectl() {
     exe kubectl config set-context "$CONTEXT_NAME" --user="$AUTH_NAME" --cluster="$CLUSTER_NAME"
 
     exe kubectl config use-context "$CONTEXT_NAME"
+
+    # Optional. Use the AWS EKS authenticator
+    local use_aws_iam_authenticator
+    use_aws_iam_authenticator="$(jq -r '.source.use_aws_iam_authenticator // ""' < "$payload")"
+    local aws_eks_cluster_name
+    aws_eks_cluster_name="$(jq -r '.source.aws_eks_cluster_name // ""' < "$payload")"
+    if [[ "$use_aws_iam_authenticator" == "true" ]]; then
+      if [ -z "$aws_eks_cluster_name" ]; then
+        echoerr 'You must specify a clustername when using aws_iam_authenticator.'
```

Comment:
Can we make this more precise? with file/variable that needs to be changed or an example even